scrapi/consumers/uiucideals/LICENSE

```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```

Comment:
Can the consumers with licenses just get the license from scrapi, or is it important that they have this specific license?